apps/linkedin-to-notion/src/components/LinkedInNotionSidePanel/Form/Form.tsx

```diff
@@ -95,29 +95,35 @@ export const Form = ({
   }, [displayNotionValues, currentNotionValues, linkedinValues]);
 
   useEffect(() => {
-    if (selectedNotionDatabase) {
-      findProfileInNotionDatabase(selectedNotionDatabase, linkedinUrl);
+    if (notionToken && selectedNotionDatabase) {
+      if (linkedinUrl) {
```

Comment:
Why not a triple "and" clause?

Reply:
Good question. Honestly I spent no time on this, I can update.